sdk/highlight-py/highlight_io/integrations/fastapi.py

```diff
@@ -35,6 +35,10 @@ async def dispatch(
                     status_code=resp.status_code,
                     headers=resp.headers.__dict__,
                     detail=body.decode(),
```

Comment:
Wondering wether its a hustle to also record the request body here and not only the response body?
Often its useful and helpful for error reproduction to understand whats the request payload.

Reply:
updated to capture the request body when possible!